package.json

```diff
@@ -109,6 +109,7 @@
     "cookie": "^0.4.0",
     "copy-template-dir": "^1.4.0",
     "debug": "^4.1.1",
+    "decache": "^4.6.0",
```

Comment:
While the repo hasn't be updated in over a year, the package has many downloads:
https://www.npmjs.com/package/decache